dynatrace_health/stackstate_checks/dynatrace_health/dynatrace_health.py

```diff
@@ -0,0 +1,272 @@
+# (C) StackState 2021
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import time
+from datetime import datetime, timedelta
+
+from schematics import Model
+from schematics.types import IntType, StringType, ListType, DictType, URLType, BooleanType, ModelType
+
+from stackstate_checks.base import StackPackInstance, HealthStream, HealthStreamUrn, Health, Identifiers
+from stackstate_checks.checks import AgentCheck
+from stackstate_checks.dynatrace.dynatrance_client import DynatraceClient
+
+VERIFY_HTTPS = True
+TIMEOUT = 10
+EVENTS_BOOSTRAP_DAYS = 5
+EVENTS_PROCESS_LIMIT = 10000
+RELATIVE_TIME = 'hour'
+ENVIRONMENT = 'production'
+DOMAIN = 'dynatrace'
+
+dynatrace_entities_cache = {}
+
+DYNATRACE_UI_URLS = {
```

Comment:
this won't be used because there is no data in the cache and then no URL will be created. good candidate to write a test for.